src/pages/index.js

```diff
@@ -31,19 +33,23 @@ const profileEditValidator = new FormValidator(
 
 const cardList = new Section(
   {
-    items: initialCards,
     renderer: (item) => {
       const cardELement = createCard(item);
       cardList.addItem(cardELement);
     },
   },
   ".cards__list"
 );
-cardList.renderItems();
+//cardList.renderItems();
+const api = new Api("https://around-api.en.tripleten-services.com/v1", {
+  authorization: "f5e7da7f-f9a4-4037-8dd1-a9066e254adc",
```

Comment:
According to the task you need to pass full `headers` into the constructor of `Api`. See here https://snipboard.io/pRdaAZ.jpg . And in the code you need to use only: `headers: this._headers` (not to duplicate the code)